### PR TITLE
Trusted proxies were removed when URL signing took over

### PR DESCRIPTION
**Pull request by rawkode**
Since: https://github.com/symfony/http-kernel/commit/fa8f4f8137444d267c087a404bc5532b14f10463

### CODE REVIEW

File: components/http_foundation/trusting_proxies.rst [hunk 1]
```diff
@@ -1,55 +0,0 @@
-.. index::
-   single: Request; Trusted Proxies
-
-Trusting Proxies
```
Comment (stof) [inline components/http_foundation/trusting_proxies.rst]: this removal is wrong. We still have trusted proxies (even though ESI accepts signed requests from anywhere)

File: cookbook/cache/varnish.rst [hunk 2]
```diff
@@ -12,17 +12,6 @@ cached content fast and including support for :ref:`Edge Side Includes <edge-sid
 .. index::
     single: Varnish; configuration
 
-Make Symfony Trust the Reverse Proxy
```
Comment (stof) [inline cookbook/cache/varnish.rst]: This removal is wrong too

File: cookbook/request/load_balancer_reverse_proxy.rst [hunk 3]
```diff
@@ -1,108 +0,0 @@
-How to Configure Symfony to Work behind a Load Balancer or a Reverse Proxy
```
Comment (stof) [inline cookbook/request/load_balancer_reverse_proxy.rst]: this must be reverted too

**Comment by stof** [comment]
the commit you linked does not remove trusted proxies at all. It removed their usage in a single place which was not working properly

**Comment by rawkode** [comment]
Sorry, @stof - silly assumption on my part. I've removed all the daft removals.

TIL more about Symfony and the FragmentListener 👍

**Comment by xabbuh** [comment]
👍

**Comment by WouterJ** [comment]
Thank you @rawkode for updating the code.

As this security fix was merged into the 2.3 version of the code, I've merged this into the 2.3 version of the docs as well. I'll take care of merging it into the other versions.

**Pull request closed without merging**